### PR TITLE
Appetize Wrapper

### PR DESCRIPTION
**Pull request by mpcarolin**
# Goals
* implement an appetize api wrapper (starting with the upload endpoint)

# Updates
* I simply refactored and wrote tests for the appetize wrapper that was based in tap-snack before

# Testing
* Clone the repo
* Create a `.env` file at the root directory
* Fill in this:
```bash
PLATFORM=ios
TOKEN=<your appetize token>
URL=https://exp-shell-app-assets.s3.us-west-1.amazonaws.com/ios/%40mpcarolin/kegcore-4ed71545-f25d-4563-83a0-4e05c302262c-simulator.tar.gz
# BUILD_PATH=<path to a local build file>
```
* Run `yarn test` -- ensure all tests pass
* Run `yarn apt:test`
  * ensure the request succeeds
* Now comment out URL in your `.env` and uncomment `BUILD_PATH`
* Run `yarn apt:test` again and verify that succeeds

### CODE REVIEW

File: scripts/apt-test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const { upload } = require('../src/api')
+
+const {
+  BUILD_PATH,
+  PLATFORM,
+  TOKEN,
+  URL,
+} = process.env
+
+if (!BUILD_PATH || !PLATFORM || !TOKEN || !URL)
```
Comment (lancetipton) [inline scripts/apt-test.js]: This should check for `!BUILD_PATH && !URL`
Technically only one of them is required

Updated to look like this and it works => 
`if ((!BUILD_PATH && !URL) || !PLATFORM || !TOKEN)`

**Review by lancetipton — CHANGES_REQUESTED** [review]
Looks good, just a couple comments:
1. Were you planning on publishing this to NPM?
2. We should include a `example.env` that includes example ENVs.
  * This way the user knows which ENVs need to be set
  * It also would be nice to add it to the readme
3. I left 1 comment about issue that should be fixed

File: scripts/apt-test.js (hunk 1 above)
Comment (mpcarolin) [reply scripts/apt-test.js]: Ah, this was a last minute update that I messed up. My bad. Thanks, it's been updated.

Comment (mpcarolin) [reply scripts/apt-test.js]: Also, yes, this is planned to be published to NPM. I just updated it to have a "repository" and a scoped "name" field, so it should be good to go once it's approved.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "demo": "node index.js",
+    "test": "yarn jest --config ./configs/jest.config.js",
+    "apt:test": "node ./scripts/apt-test.js"
+  },
+  "dependencies": {
+    "@keg-hub/jsutils": "8.6.0",
+    "axios": "^0.21.1",
+    "form-data": "^4.0.0",
+    "rimraf": "3.0.2",
+    "tar-fs": "2.1.1"
```
Comment (lancetipton) [inline package.json]: Not blocking but I'm pretty sure this is not used, I could be wrong though

**Review by lancetipton — APPROVED** [review]
Looks good. Left one comment, but it's not blocking

File: package.json (hunk 2 above)
Comment (mpcarolin) [reply package.json]: updated

**Pull request merged**